docs/README.md

```diff
@@ -0,0 +1,80 @@
+# Getting Started
+
+# Pre-requisites
+
+* [Python 3.6.0](https://www.python.org/downloads/release/python-360/)+ (including `pip`)
+* [Docker](https://www.docker.com/) (only needed for `DockerAgent`)
+* [virtualenv](https://virtualenv.pypa.io/en/stable/) (optional, for isolated Python environment)
+
+# Installation
+
+* **OPTIONAL**: Setup an isolated virtual Python environment by running the following commands
+```
+$ virtualenv </path/to/venv>
+```
+This environment needs to be activated for usage. Any package installations will now persist
+in this virtual environment folder only.
+```
+source venv/bin/activate
+```
+
+* Clone the repository
+```
+$ git clone https://github.com/MultiAgentLearning/playground </path/to/playground>
+```
+
+* Install the `pommerman` package. This needs to be done every time the code is updated to get the
+latest modules
+```
+$ cd </path/to/playground>
+$ pip install -U .
+```
+
+# Examples
+
+**NOTE**: There can only be **four** agents in the game.
+
+## A Simple Example
+
+A simple game run in a Free-For-All configuration can be found in [simple_ffa_run.py](../examples/simple_ffa_run.py) which
+places two [SimpleAgent](../pommerman/agents/simple_agent.py)s and two [RandomAgent](../pommerman/agents/random_agent.py)s
+on the board.
+
```

Comment:
The above can be simplied:
"Use simple_ffa_run.py to run an example Free-For-All with two SimpleAgents and two RandomAgents."